app/entry.server.tsx

```diff
@@ -1,28 +1,56 @@
-import { renderToString } from "react-dom/server";
+import { PassThrough } from "node:stream";
+import { renderToPipeableStream } from "react-dom/server";
```

Comment:
Kan vi få dette til med [`renderToReadableStream`](https://reactjs.org/docs/react-dom-server.html#rendertoreadablestream) i stedet?

Remix støtte for ReadableStream: https://github.com/remix-run/remix/releases/tag/v1.5.0

Reply:
Du må gjerne prøve, men jeg får det ikke til.. Sikker på at den funker i node? Jeg får ikke eksempel-repoet opp å kjøre en gang, og de bruker i tillegg eksperimentelle versjoner av alle remix-avhengighetene sine 😬 https://github.com/netlify/remix-edge-template/blob/main/package.json#L14

Reply:
Fikk samme, og denne tweeten ligger lenger nede i tråden: https://twitter.com/ebey_jacob/status/1516899093599059968?s=20&t=o1ivQ9ra21vdS97OjsrFHA

Var den linken der jeg fikk `renderToPipeableStream`-implementasjonen min fra 😅